SETUP/CHANGELOG.md

```diff
@@ -3,13 +3,34 @@
 Major changes to this project are documented here. For minor changes,
 see the git history.
 
-## R??????
+## R202102
 Scripts supporting this upgrade are in `SETUP/upgrade/15`
 
 **If you are upgrading from a release before R202009, you must upgrade to
 [R202009](https://github.com/DistributedProofreaders/dproofreaders/releases/tag/R202009)
 first before upgrading to this or later releases.**
 
+**This is the last release to support PHP versions < 7.4 and
+MySQL versions < 5.7.**
+
+* Support MySQL 8.0, PHP 7.2, and PHP 7.4 (chrismiceli, cpeel, bfoley)
+* New RESTful API - see [API.md](API.md) (cpeel, bfoley)
+* New user workflow improvements allows P1 to have a quiz requirement (cpeel)
+* Project comments can be either HTML or Markdown (chrismiceli)
+* Format Preview enhancements, including LaTeX math preview (70ray)
+* New dark theme: Charcoal (srjfoo)
+* Special Days projects identified by emojis in round listings (cpeel)
+* PPVers can return a project back to a PP (70ray)
```

Comment:
I think this should be "PPers can return a project back to the PPVer who returned it to them (70ray)" (Also -- are you using DP usernames or github usernames?)

Reply:
Thanks, I will get this updated.

In the changelog we use the github usernames when possible (bunny-crunch being the primary exception in this list I think). Oh wait, I have Brian's username wrong too -- should be bpfoley. I'll fix that too.

Reply:
Updated.